packages/prisma-client/prisma/schema.prisma

```diff
@@ -81,6 +83,10 @@ model Build {
 
 model Tree {
   id              String @id @default(uuid())
```

Comment:
See #896 I think we need to remove `@id`  attribute here and add `@@id([id, projectId])`, Sorry I totally forgot about this in issues.

Reply:
Will default work with it?